src/webportal/src/app/job/job-view/fabric/task-attempt/task-attempt-list.jsx

```diff
@@ -164,40 +208,63 @@ export default class TaskAttemptList extends React.Component {
     }
   }
 
-  showContainerLog(logUrl, logType) {
-    let title;
-    let logHint;
-
-    if (config.logType === 'yarn') {
-      logHint = 'Last 4096 bytes';
-    } else if (config.logType === 'log-manager') {
-      logHint = 'Last 16384 bytes';
-    } else {
-      logHint = '';
-    }
-    switch (logType) {
-      case 'stdout':
-        title = `Standard Output (${logHint})`;
-        break;
-      case 'stderr':
-        title = `Standard Error (${logHint})`;
-        break;
-      case 'stdall':
-        title = `User logs (${logHint}. Notice: The logs may out of order when merging stdout & stderr streams)`;
-        break;
-      default:
-        throw new Error(`Unsupported log type`);
+  convertObjectFormat(logUrls) {
+    const logs = {};
+    for (const p in logUrls.locations) {
+      logs[logUrls.locations[p].name] = logUrls.locations[p].uri;
     }
-    this.setState(
-      {
-        monacoProps: { value: 'Loading...' },
-        monacoTitle: title,
-        logUrl,
-      },
-      () => {
-        this.logAutoRefresh(); // start immediately
-      },
-    );
+    return logs;
+  }
+
+  showAllLogDialog(logListUrl) {
+    const { hideAllLogsDialog } = this.state;
+
+    getContainerLogList(logListUrl).then(({ fullLogUrls, _ }) => {
+      this.setState({
+        hideAllLogsDialog: !hideAllLogsDialog,
+        fullLogUrls: fullLogUrls,
+      });
+    });
```

Comment:
Forget `.catch(() => this.setState({ hideAllLogsDialog: !hideAllLogsDialog }));` fixed in #5103

Reply:
Fixed, thx.